packages/discovery-provider/src/queries/query_helpers.py

```diff
@@ -24,19 +31,24 @@
 from src.models.users.user import User
 from src.models.users.user_bank import UserBankAccount
 from src.queries import response_name_constants
+from src.queries.get_authorization import is_authorized_request
 from src.queries.get_balances import get_balances
 from src.queries.get_track_comment_count import get_tracks_comment_count
 from src.queries.get_unpopulated_users import get_unpopulated_users
 from src.trending_strategies.trending_type_and_version import TrendingVersion
 from src.utils import helpers, redis_connection
+from src.utils.get_all_nodes import get_all_healthy_content_nodes_cached
+from src.utils.rendezvous import RendezvousHash
 
 logger = logging.getLogger(__name__)
 
 redis = redis_connection.get_redis()
 
 
 # ####### VARS ####### #
-
+# How many nodes a file is replicated to
+# See: https://github.com/AudiusProject/audius-protocol/blob/3d223085b07b53073c475992f54315ae826dc91d/pkg/mediorum/mediorum.go#L125
+REPLICATION_FACTOR = 4
```

Comment:
could either:
1. keep this in sync with actual replica count
2. choose arbitrary number (CN redirects to a node that would have the content anyway)

Reply:
for images, for now, we elected to do 2 replicas, so let's keep that in sync here and do 2?